app/serializers/api/v1/participant_declaration_serializer.rb

```diff
@@ -34,6 +34,12 @@ class ParticipantDeclarationSerializer
       attribute :state do |declaration|
         declaration.current_state.dasherize
       end
+
+      attribute :has_passed, if: FeatureFlag.active?(:participant_outcomes_feature) do |declaration|
+        if declaration.npq?
+          declaration.outcomes.latest&.passed?
```

Comment:
when latest is empty, we are getting an empty association, might need to amend this scope

Reply:
@cwrw 
Sure, its the reason why i changed `latest` to a class method instead of scope.